frontend/app/src/locales/en.json

```diff
@@ -35,9 +35,14 @@
     "label_username": "Username",
     "label_password": "Password",
     "sync_error": {
-      "title": "Sync Error",
+      "title": "Sync database mismatch",
       "button_yes": "Yes",
-      "button_no": "No"
+      "button_no": "No",
+      "message": "A remote database was detected BUT with an older last modification timestamp that the local one. {0}",
```

Comment:
Even though this should not be here, there is a typo. that the -> than the

Reply:
> @kelsos
> All right so I think two things are missing here.
> 
>     1. The sync error message is not hard coded and should be taken from the result['message']
> 
>     2. At the very end of the list of changes. There should be a question: "Would you like to replace your local DB with the remote one?"

I can try that, but we have to start thinking about hardcoded messages in the backend and how we display them. Also I have to verify once again that this was not a problem with the parsing, but when I tried to use the backend's message I saw something that was `"('message', { payload...})"`

Reply:
> we have to start thinking about hardcoded messages in the backend and how we display them

Out of scope for now. This would be a big change.